src/User/src/Adapter/AuthenticationAdapter.php

```diff
@@ -72,30 +51,18 @@ public function authenticate(): Result
             );
         }
 
-        /** Check for extra validation options */
         if (! empty($this->config['options'])) {
             foreach ($this->config['options'] as $property => $option) {
                 $methodName = 'get' . ucfirst($property);
 
-                /** Check if $methodName exists in the provided identity class */
                 $this->checkMethod($identityClass, $methodName);
 
-                /** Check if value for the current option is provided */
-                if (! array_key_exists('value', $option)) {
-                    throw new Exception(sprintf(
-                        self::OPTION_VALUE_NOT_PROVIDED,
-                        'value',
-                        $property
-                    ));
+                if (empty($option['value'])) {
```

Comment:
```suggestion
                if (! array_key_exists('value', $option)) {
```
We need to keep the `array_key_exists` check because `$option['value']` can have an empty value, see [this](https://github.com/dotkernel/frontend/blob/4.0/config/autoload/authentication.global.php#L27) example.